Unsafe load() call disabled by Gentoo.

Seems that gentoo have disable yaml.load for security reasons (see https://bugs.gentoo.org/659348 ) and buku tests fails on gentoo. Could we use safe_load on buku? Seems that it's not as easy as replacing "load" to "safe_load".

Extract from tests logs

```
=================================================================================================================================== FAILURES ====================================================================================================================================
________________________________________________________________________________________________________________________ test_load_chrome_database[True] ________________________________________________________________________________________________________________________

chrome_db = ('/tmp/portage/www-misc/buku-4.2.2/work/buku-4.2.2/tests/test_bukuDb/Bookmarks', '/tmp/portage/www-misc/buku-4.2.2/wor...bukuDb/25491522_res.yaml', '/tmp/portage/www-misc/buku-4.2.2/work/buku-4.2.2/tests/test_bukuDb/25491522_res_nopt.yaml'), add_pt = True

    @pytest.mark.parametrize('add_pt', [True, False])
    def test_load_chrome_database(chrome_db, add_pt):
        """test method."""
        # compatibility
        json_file = chrome_db[0]
        res_yaml_file = chrome_db[1] if add_pt else chrome_db[2]
        dump_data = False  # NOTE: change this value to dump data
        if not dump_data:
            with open(res_yaml_file, 'r') as f:
>               res_yaml = yaml.load(f)

tests/test_bukuDb.py:1321:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

stream = <_io.TextIOWrapper name='/tmp/portage/www-misc/buku-4.2.2/work/buku-4.2.2/tests/test_bukuDb/25491522_res.yaml' mode='r' encoding='UTF-8'>, Loader = None

    def load(stream, Loader=None):
        """
        Parse the first YAML document in a stream
        and produce the corresponding Python object.
        """
        if Loader is None:
>           raise RuntimeError("Unsafe load() call disabled by Gentoo. See bug #659348")
E           RuntimeError: Unsafe load() call disabled by Gentoo. See bug #659348

/usr/lib64/python3.6/site-packages/yaml/__init__.py:109: RuntimeError
```

--- a/tests/test_bukuDb.py
+++ b/tests/test_bukuDb.py
@@ -1318,7 +1318,10 @@ def test_load_chrome_database(chrome_db, add_pt):
     dump_data = False  # NOTE: change this value to dump data
     if not dump_data:
         with open(res_yaml_file, 'r') as f:
-            res_yaml = yaml.load(f, Loader=yaml.FullLoader)
+            try:
+                res_yaml = yaml.load(f, Loader=yaml.FullLoader)
+            except RuntimeError:
+                res_yaml = yaml.safe_load(f)
     # init
     import buku
     bdb = buku.BukuDb()
@@ -1359,7 +1362,10 @@ def test_load_firefox_database(firefox_db, add_pt):
     res_yaml_file = firefox_db[1] if add_pt else firefox_db[2]
     if not dump_data:
         with open(res_yaml_file, 'r') as f:
-            res_yaml = yaml.load(f)
+            try:
+                res_yaml = yaml.load(f)
+            except RuntimeError:
+                res_yaml = yaml.safe_load(f)
     # init
     import buku
     bdb = buku.BukuDb()

